CHANGELOG.md

```diff
@@ -231,6 +231,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Removed `teardown` from `ParallelPlugin` ([#8943](https://github.com/PyTorchLightning/pytorch-lightning/pull/8943))
 
 
+- Removed deprecated `profiled_functions` argument from `PyTorchProfiler` ([#9178](https://github.com/PyTorchLightning/pytorch-lightning/pull/9178))
```

Comment:
```suggestion
- Removed deprecated `profiled_functions` and `record_functions` arguments from `PyTorchProfiler` ([#9178](https://github.com/PyTorchLightning/pytorch-lightning/pull/9178))
```

Reply:
@Borda The `record_functions` argument doesn't seem to be deprecated or due for removal. Did I miss something?

Reply:
you were removing the warning which was referring to it...